res/css/views/settings/_UserProfileSettings.pcss

```diff
@@ -27,6 +27,7 @@ limitations under the License.
         .mx_UserProfileSettings_profile_displayName {
             flex-grow: 1;
             width: 100%;
+            gap: 0;
```

Comment:
Not really related but this is another thing that is specified at the dialog level and we need to override, as it's giving us extra spacing.